CemEzgin/hw3/scripts/script.js

```diff
@@ -0,0 +1,73 @@
+equal.addEventListener('click',takeToArray);
```

Comment:
eşittir butonuna basıldığı zaman işlemi yapması için HTML de eşittir butonunun id si.